src/components/base/Select/Select.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
```suggestion
import React from 'react';
```
이 부분도 사용되지 않고 있다고 합니다!